var/spack/repos/builtin/packages/nvhpc/package.py

```diff
@@ -341,6 +341,7 @@ class Nvhpc(Package):
     )
     variant("lapack", default=True, description="Enable LAPACK")
     variant("mpi", default=False, description="Enable MPI")
+    variant("cuda_version", default="default", description="CUDA version, for example 11.8")
```

Comment:
Not a blocking comment, but should this rather be called:
```suggestion
    variant("default_cuda", default="default", description="CUDA version, for example 11.8")
```
since it doesn't actually force a particular version?

Reply:
I also like `default_version` better.

Reply:
resolved